hw/top_darjeeling/doc/datasheet.md

```diff
@@ -154,7 +154,7 @@ Contributions are welcome, but this design is provided as-is.
 - The binary translation logic and the SoC DFx control module are under development. The former will allow for granular filtering and remapping of TL-UL requests, while the latter will provide SoC level gating functionality for DFT and debug infrastructure.
 - Crossbar generation support at the SoC-integration is not yet mature, so some TL-UL connections have not been made at this point.
 
-Since Darjeeling was derived from the discrete "Earl Grey" chip, several legacy structures are temporarily present.
+Since Darjeeling was derived from the discrete "Earl Grey" chip, several legacy structures currently remain.
```

Comment:
We want to emphasise that these are temporary. Could say temporarily remain?

Reply:
That works for me